nav_msgs/Path path
int8[] direction